### PR TITLE
[IMP] mail: add lang to default value for suggested reciptient (wip)

### PR DESCRIPTION
**Pull request by pko-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/models/suggested_recipient_info/suggested_recipient_info.js [hunk 1]
```diff
@@ -68,6 +75,9 @@ function factory(dependencies) {
          * Determines the optional `mail.partner` associated to `this`.
          */
         partner: many2one('mail.partner'),
+        lang: attr({
```
Comment (seb-odoo) [inline addons/mail/static/src/models/suggested_recipient_info/suggested_recipient_info.js:78]: guideline: sort fields alphabetically and add docstring

File: addons/mail/static/src/models/suggested_recipient_info/suggested_recipient_info.js [hunk 2]
```diff
@@ -37,6 +37,13 @@ function factory(dependencies) {
             return this.partner && this.partner.nameOrDisplayName || this.name;
         }
 
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeLang() {
+            return this.partner && this.partner.lang || this.lang;
+        }
```
Comment (seb-odoo) [inline addons/mail/static/src/models/suggested_recipient_info/suggested_recipient_info.js:46]: Better not make this adjusting compute, I don't think you actually need the value once the partner is created (and if you do, you can access it through the partner anyway).

Just keep the lang of the suggested recipient intact, which is the lang originally given by the python for this suggestion.

**Review by seb-odoo — COMMENTED** [review]
I see it's wip but as I saw it anyway, small comments below.

**Comment by awa-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**